src/index.js

```diff
@@ -30,7 +30,9 @@ import UserWebBrowser from './user-web-browser';
 import Onboarding from './ui/Onboarding';
 import { WebBrowserConsumer } from './ui/WebBrowser';
 
-// need to initialize bugsnag at the entrypoint
+// need to initialize error tracking at the entrypoint
+// eslint-disable-next-line
+import Crashes from 'appcenter-crashes';
```

Comment:
@redreceipt We don't need to do anything with this? Just import it and we auto get crash info?

Reply:
I think so